modules/setting/setting.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/gogits/gogs/modules/bindata"
 	"github.com/gogits/gogs/modules/log"
 	"github.com/gogits/gogs/modules/user"
+
+	"strk.kbt.io/projects/go/libravatar"
```

Comment:
Vendoring would minimize them even further, w/out stamping a
proprietary software based service brand on top of free software
developers works